hlx_statics/scripts.js

```diff
@@ -1,17 +1,64 @@
 let $IS_HLX_PATH = false;
 let $IS_STAGE = false;
+let $IS_DEV = false;
 
-if(window.location.host.indexOf('hlx.page') >= 0 || window.location.host.indexOf('hlx.live') >= 0 || window.location.host.indexOf('localhost') >= 0){
+if (window.location.host.indexOf('hlx.page') >= 0 || window.location.host.indexOf('hlx.live') >= 0 || window.location.host.indexOf('localhost') >= 0) {
   $IS_HLX_PATH = true;
 }
 
-if(window.location.host.indexOf('stage.adobe.io') >= 0 || window.location.host.indexOf('developer-stage') >= 0){
+if (window.location.host.indexOf('stage.adobe.io') >= 0 || window.location.host.indexOf('developer-stage') >= 0) {
   $IS_STAGE = true;
 }
 
+if (window.location.host.indexOf('localhost') >= 0) {
+  $IS_DEV = true;
+}
+
```

Comment:
I added a separate DEV flag since I needed that utility. Can I remove the localhost check from HLX_PATH conditional?